### PR TITLE
feat(frontend/paddle/loss): implement paddle square error cost. Closes #21862

### LINKED ISSUE #21862: square_error_cost

### PR DESCRIPTION
**Pull request by asuzukosi**
Closes #21862 
Implement the square error cost function as part of ivy job assessment task.
**procedures**
- Followed the loss function implementation highlighted in the paddle documentation [here](https://www.paddlepaddle.org.cn/documentation/docs/en/api/paddle/nn/functional/square_error_cost_en.html)
- In the loss file for the paddle frontend on line *75* added the square error loss
- In the loss function test file on paddle frontend on line *108* add the test for the square error loss function
- Followed the writing procedure for previously implemented loss functions

**future work**
- Some the the loss functions in the paddle file are still using manual if else checks for reduction selection, this should be replaced with the `_get_reduction_function` in all locations where the repeated logic is found. 

**available for corrections and modifications**

### CODE REVIEW

**Comment by asuzukosi** [comment]
close #21862

**Comment by asuzukosi** [comment]
@bipinKrishnan Please can you review my PR?

**Comment by asuzukosi** [comment]
Okay, thank you @bipinKrishnan

**Comment by asuzukosi** [comment]
Hello, @bipinKrishnan , wanted to ask if there is any updates with the PR resolution

**Comment by bipinKrishnan** [comment]
Hey @asuzukosi , it would be nice if you could install and configure pre-commit in your system to avoid lint test fails.

**Comment by bipinKrishnan** [comment]
ivy-gardener

**Comment by bipinKrishnan** [comment]
> Hello, @bipinKrishnan , wanted to ask if there is any updates with the PR resolution

I have linked your issue to the main to-do list.

**Comment by asuzukosi** [comment]
@bipinKrishnan I have merged the changes from Ivy gardener into my branch. Let me know if any other changes are needed, thank you.

**Comment by bipinKrishnan** [comment]
ivy-gardener

File: ivy/functional/frontends/paddle/nn/functional/loss.py [hunk 1]
```diff
@@ -428,6 +428,16 @@ def softmax_with_cross_entropy(
     return paddle.to_tensor(loss)
 
 
+@handle_exceptions
+@to_ivy_arrays_and_back
+@with_supported_dtypes({"2.4.2 and below": ("float32", "float64")}, "paddle")
+def square_error_cost(input, label, name=None):
```
Comment (bipinKrishnan) [inline ivy/functional/frontends/paddle/nn/functional/loss.py:434]: Hey, there is another function in line 443 with the same name. Why is this implemented again? Let me know if I missed something.

**Pull request closed without merging**